packages/grafana-schema/src/scuemata/dashboard/dist/family.cue

```diff
@@ -27,6 +28,7 @@ import (
 Family: dashboard.Family & {
     compose: Panel: {
         // TODO do this with a loop once we include the panel type/plugin id in the model
+        annolist: pannolist.Panel
```

Comment:
FYI - this is where we compose panel plugin schemas into the main dashboard schemas. This is all the old scuemata system, and will be removed and replaced by something similar when i finish grafana/thema#8

(tests caught this one, i'd forgotten about it myself while we were mobbing)